### PR TITLE
Add test for building of package

### PR DESCRIPTION
**Pull request by sevazhidkov**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,4 +2,6 @@ language: python
 python:
   - "2.7"
 install: "pip install -r requirements.txt"
-script: python test.py
+script:
+- python test.py
+- python setup.py install
```
Comment (sudheesh001) [inline .travis.yml]: Can you add `- python setup.py build` before the `install` step, then this is good to merge.

**Comment by sevazhidkov** [comment]
Done.

**Comment by sudheesh001** [comment]
@sevazhidkov I still don't see the changes here.

**Comment by sevazhidkov** [comment]
Oh, I understood your. ```python setup.py install``` and ```python setup.py build``` is equivalent for tests: command creating virtual build.

 ```install``` command checking even more that ```build```.

Install:
```
running install
Checking .pth file support in /Library/Python/2.7/site-packages/
/usr/bin/python -E -c pass
TEST PASSED: /Library/Python/2.7/site-packages/ appears to support .pth files
running bdist_egg
running egg_info
creating python_loklak_api.egg-info
writing requirements to python_loklak_api.egg-info/requires.txt
writing python_loklak_api.egg-info/PKG-INFO
writing top-level names to python_loklak_api.egg-info/top_level.txt
writing dependency_links to python_loklak_api.egg-info/dependency_links.txt
writing entry points to python_loklak_api.egg-info/entry_points.txt
writing manifest file 'python_loklak_api.egg-info/SOURCES.txt'
reading manifest file 'python_loklak_api.egg-info/SOURCES.txt'
writing manifest file 'python_loklak_api.egg-info/SOURCES.txt'
installing library code to build/bdist.macosx-10.11-intel/egg
running install_lib
running build_py
creating build/bdist.macosx-10.11-intel
creating build/bdist.macosx-10.11-intel/egg
copying build/lib/loklak.py -> build/bdist.macosx-10.11-intel/egg
byte-compiling build/bdist.macosx-10.11-intel/egg/loklak.py to loklak.pyc
creating build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying python_loklak_api.egg-info/PKG-INFO -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying python_loklak_api.egg-info/SOURCES.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying python_loklak_api.egg-info/dependency_links.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying python_loklak_api.egg-info/entry_points.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying python_loklak_api.egg-info/not-zip-safe -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying python_loklak_api.egg-info/requires.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying python_loklak_api.egg-info/top_level.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
creating dist
creating 'dist/python_loklak_api-1.0-py2.7.egg' and adding 'build/bdist.macosx-10.11-intel/egg' to it
removing 'build/bdist.macosx-10.11-intel/egg' (and everything under it)
Processing python_loklak_api-1.0-py2.7.egg
removing '/Library/Python/2.7/site-packages/python_loklak_api-1.0-py2.7.egg' (and everything under it)
creating /Library/Python/2.7/site-packages/python_loklak_api-1.0-py2.7.egg
Extracting python_loklak_api-1.0-py2.7.egg to /Library/Python/2.7/site-packages
python-loklak-api 1.0 is already the active version in easy-install.pth
Installing python-loklak-api script to /usr/local/bin

Installed /Library/Python/2.7/site-packages/python_loklak_api-1.0-py2.7.egg
Processing dependencies for python-loklak-api==1.0
Searching for wsgiref==0.1.2
Best match: wsgiref 0.1.2
wsgiref 0.1.2 is already the active version in easy-install.pth

Using /System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7
Searching for requests==2.8.1
Best match: requests 2.8.1
Processing requests-2.8.1-py2.7.egg
requests 2.8.1 is already the active version in easy-install.pth

Using /Library/Python/2.7/site-packages/requests-2.8.1-py2.7.egg
Searching for asyncoro==3.5
Best match: asyncoro 3.5
Processing asyncoro-3.5-py2.7.egg
asyncoro 3.5 is already the active version in easy-install.pth
Installing discoro.py script to /usr/local/bin
Installing discoronode.py script to /usr/local/bin

Using /Library/Python/2.7/site-packages/asyncoro-3.5-py2.7.egg
Finished processing dependencies for python-loklak-api==1.0
```

Build:
```
running build
running build_py
creating build
creating build/lib
```

**Comment by sudheesh001** [comment]
Okay sounds good. Fair justification. :smile:

**Pull request merged**